src/quo/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -15,7 +15,7 @@
 (defn- view-internal
   []
   (let [pressed? (reagent/atom false)]
```

Comment:
should we add a schema to this component while we are adding new props to it? 
I think it's not in the list of components in the schema epic as I probably missed this ns when making those issues

Reply:
@J-Son89 - Can we do it in a separate PR/issue? To find and fix bugs related to those changes (if any).

Reply:
Of course!